crates/router/src/core/payment_methods/transformers.rs

```diff
@@ -15,12 +15,25 @@ use crate::{
     utils::{self, OptionExt},
 };
 
+#[derive(Debug, Serialize)]
+#[serde(untagged)]
+pub enum StoreLockerReq<'a> {
+    LockerCard(StoreCardReq<'a>),
+    LockerGeneric(StoreGenericReq<'a>),
+}
+
 #[derive(Debug, Deserialize, Serialize)]
 pub struct StoreCardReq<'a> {
     pub merchant_id: &'a str,
     pub merchant_customer_id: String,
     pub card: Card,
-    pub enc_card_data: Option<String>,
+}
+
+#[derive(Debug, Deserialize, Serialize)]
+pub struct StoreGenericReq<'a> {
+    pub merchant_id: &'a str,
+    pub merchant_customer_id: String,
+    pub enc_card_data: String,
```

Comment:
Will this be `enc_card_data`, we are planning to use this to store any type of payment method right? can we have a bit of a generic name